client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableStyles.ts

```diff
@@ -29,6 +29,11 @@ const useStyles = (isWide: boolean) => makeStyles((theme: Theme) => ({
         paddingTop: theme.spacing(1),
         paddingBottom: theme.spacing(1),
         width: tableWidth,
+        justifyContent: 'space-between'
+    },
+    titleText: {
```

Comment:
Class is redundant , use alignItems='center' on the grid itself

Reply:
The flex must be applied, but added alignItems to the code